boards/arm/fanstel_ev832x/doc/fanstel_ev832x.rst

```diff
@@ -0,0 +1,23 @@
+.. _fanstel_ev832x:
+
+Fanstel EV-BT832X
+#################
+
+Overview
+********
+
+The Fanstel EV-BT832X board is an evaluation board for the BluNor BT832X
+Bluetooth Low Energy module. This board uses Nordic Semiconductor nRF52832 SoC,
+with an ARM Cortex (TM) M4F MCU, available 512KB flash, 64KB RAM, embedded
+2.4GHz multi-protocol transceiver, power amplifier, and an integrated PCB trace
+antenna, or an u.FL connector for external antenna.
+
+See `fanstel ev-bt832x website`_ for more information about the development
+board and `nRF52832 website`_ for the official reference on the IC itself.
+
```

Comment:
Good so are... but still missing board usage information typical of our board docs (see, for example, https://docs.zephyrproject.org/latest/boards/arm/nucleo_l053r8/doc/nucleol053r8.html)